docs/data/base/getting-started/customization/customization.md

```diff
@@ -81,3 +115,84 @@ Use them to apply styling to your HTML elements.
 The `getInputProps` function can be used to get the props to place on an HTML `<input>` to make the switch accessible.
 
 {{"demo": "StylingHooks.js", "defaultCodeOpen": true}}
+
+## Which option to choose?
```

Comment:
Have you considered placing this section at the top of the page immediately after the introduction?

I think the order seems more intuitive this way:
1. introduction (there are several methods...)
2. which option should I choose?
3. read about option 1|2|3|4 in depth

Reply:
Alternatively, maybe an extra sentence in the beginning of each "method of customization" section could work too, e.g.
```
## Customizing slots' props

:::info
This is the recommended method for most cases if you are using Tailwind CSS, CSS Modules, Sass, Less, or anything that compiles to CSS.
:::
```

Reply:
@michaldudak I'm inclined to agree with @mj12albert on rearranging the sections this way. If I take on the perspective of a new user, my first question after reading the intro is "ok, so which one should I choose?" I don't want to have to read each section all the way through before narrowing down my options. It also makes more sense for the links in the "which one to choose?" to link to section further down the page, rather than back up.